Sources/Sentry/include/SentryAppStartMeasurement.h

```diff
@@ -0,0 +1,32 @@
+#import "SentryDefines.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+static NSString *const SentryAppStartTypeWarm = @"warm";
+static NSString *const SentryAppStartTypeCold = @"cold";
+static NSString *const SentryAppStartTypeUnkown = @"unknown";
```

Comment:
M: should this be part of the enum? is this to avoid null checks instead in ObjC?